src/views/results/resultSetPanelProvider.ts

```diff
@@ -112,7 +112,8 @@ export class ResultSetPanelProvider implements WebviewViewProvider {
                   columnHeadings: Configuration.get(`resultsets.columnHeadings`) || 'Name',
                   queryId: this.currentQuery.getId(),
                   update_count: queryResults.update_count,
-                  isDone: queryResults.is_done
+                  isDone: queryResults.is_done,
+                  executionTime: (endTime - startTime).toFixed()
```

Comment:
So this is being sent for every request (e.g. the first request and the following page requests).

We actually only care about the execution time of the `execute` call and not all of them.

You will have to add some code to only send the execution time for the first execute (perhaps you base it on `getState()`), and `undefined` for the following requests. The frontend will need to be updated to account for this.